common/constants/shared.ts

```diff
@@ -93,6 +93,7 @@ export const PPL_INDEX_INSERT_POINT_REGEX = /(search source|source|index)\s*=\s*
 export const PPL_INDEX_REGEX = /(search source|source|index)\s*=\s*([^|\s]+)/i;
 export const PPL_WHERE_CLAUSE_REGEX = /\s*where\s+/i;
 export const PPL_NEWLINE_REGEX = /[\n\r]+/g;
+export const PPL_DESCRIBE_INDEX_REGEX = /(describe)\s*([^|\s]+)/i;
```

Comment:
Should we change `\s*` to `\s+`? (current regex would match with `describeXXX`)

Reply:
That's right, I had `\s*` because `source = <index>` allowed no whitespace but upon checking the ppl API it seems like no whitespace wouldn't work for `describe<index>`, thanks for the catch!